CHANGES.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+# 0.13.6
+
+### Bugfixes
+
+- Fixed a bug in the `k8s_job_executor` and `docker_executor` that could result in jobs exiting as `SUCCESS` before all ops have run.
+- Fixed a bug in the `k8s_job_executor` and `docker_executor` that could result in jobs failing when an op is skipped.
+
+### Dependencies
+
+- `graphene` is temporarily pinned under version 3 to unbreak Dagit dependencies.
```

Comment:
'pinned under' might be slightly confusing - 'pinned to be less than version 3'?